Sveriges radio.js

```diff
@@ -0,0 +1,709 @@
+{
+	"translatorID": "caa8f42c-9dbf-446e-963b-6ee18e3133d2",
+	"label": "Sveriges radio",
+	"creator": "Sebastian Berlin",
+	"target": "^https?://sverigesradio\\.se/sida/artikel.aspx",
```

Comment:
This doesn't seem right -- it wouldn't detect multiples if your code below is correct (please add some multiples as tests, if possible)

Reply:
Extended regex to include search page.